src/tree/box.js

```diff
@@ -88,7 +95,24 @@ export default function box(
         selections.select(node, sel, selectionState);
       }
     })
-    .html(d => card(d.data, cardStyling, sel, selectionState));
+    .html(d => card(d.data, cardStyling, sel, selectionState))
+    .on('mouseenter', d => {
+      if (allowInteractions) {
+        tooltip
+          .html(`${d.data.attributes.label || d.data.id}<br />${d.data.attributes.subLabel || ''}<br />${d.data.attributes.extraLabel || ''}<br />${d.data.measure || ''}`)
+          .attr('style', () => getTooltipStyle(d));
+      }
+    })
+    .on('mouseleave', () => {
+      tooltip
+        .html('')
+        .attr('style', 'visibility: hidden;opacity: 0;');
+    })
+    .on('mousedown', () => {
+      tooltip
+        .html('')
+        .attr('style', 'visibility: hidden;opacity: 0;');
+    });
```

Comment:
This does not interfere with zoom/panning or making selections?

Reply:
This is mostly so that the tooltip will close when panning, though they can reopen, so it isn't very effective. The panning and selections do not seem to be affected.